aggregate-data-by-county.py

```diff
@@ -0,0 +1,38 @@
+import csv
+import json
+
+data = {}
+county_index = 15
+ac_kwh_index = 8
+
+jsonreader = ""
+
+with open('data/report-3-census-tract-rev.csv', "r") as csvfile:
+    datareader = csv.reader(csvfile)
+    for row in datareader:
+
+        raw_county_name = row[county_index]
```

Comment:
this works, though I find it a bit more convenient to use `csv.DictReader` to reference fields by their name instead of index. you can see how i do it here: https://github.com/chihacknight/il-solar-map/blob/main/prep_pivot.py#L12-L15